postgres/datadog_checks/postgres/util.py

```diff
@@ -557,6 +557,23 @@ def trim_leading_set_stmts(sql):
 """.strip(),
 }
 
+QUERY_PG_WAIT_EVENT_METRICS = {
+    'name': 'wait_event_stats',
+    'columns': [
+        {'name': 'app', 'type': 'tag'},
+        {'name': 'db', 'type': 'tag_not_null'},
+        {'name': 'user', 'type': 'tag_not_null'},
+        {'name': 'wait_event', 'type': 'tag'},
+        {'name': 'backend_type', 'type': 'tag'},
+        {'name': 'activity.wait_event', 'type': 'gauge'},
+    ],
+    'query': """
+SELECT application_name, datname, usename, COALESCE(wait_event, 'NoWaitEvent'), backend_type, COUNT(*)
```

Comment:
We normally show `CPU` when there's no wait event. Shall we do the same here instead of `NoWaitEvent`